uo_sys_framework/ACREModule/cfgAttributeCategories/Object.hpp

```diff
@@ -94,5 +94,77 @@ class UO_FW_ACREOptions {
+			defaultValue = "false";
+		};
+		class UO_FW_ACRE_RACK_TYPE {
+			property = "UO_FW_ACRE_RACK_TYPE";
+			displayName = "Type of Rack";
+			tooltip = "Determines what kind of rack to add.";
+			control = "UO_FW_ACRE_Rack_Type";
+			expression = "_this setVariable ['%s',_value,true];";
+			condition = "objectVehicle + (object * (1-objectBrain)) + (1 - objectControllable)";
+			defaultValue = "'""ACRE_VRC103""'";
+		};
+		class UO_FW_ACRE_NAMES {
+			property = "UO_FW_ACRE_NAMES";
+			displayName = "Add Rack Names";
+			tooltip = "Gives racks a custom Long and short name. Short name must be four or less characters.";
+			control = "EditShort";
+			expression = "_this setVariable ['%s',_value,true];";
+			condition = "objectVehicle + (object * (1-objectBrain)) + (1 - objectControllable)";
+			defaultValue = "'[""Custom Rack Name"",""Rack""]'";
+		};
+		class UO_FW_ACRE_MOUNTED_RADIO {
+			property = "UO_FW_ACRE_MOUNTED_RADIO";
+			displayName = "Add Mounted Radio";
+			tooltip = "Add a radio to the rack by default.";
+			control = "CheckBox";
+			expression = "_this setVariable ['%s',_value,true];";
+			condition = "objectVehicle + (object * (1-objectBrain)) + (1 - objectControllable)";
+			defaultValue = "false";
+		};
+		class UO_FW_ACRE_MOUNTED_RADIO_REMOVABLE {
+			property = "UO_FW_ACRE_MOUNTED_RADIO_REMOVABLE";
+			displayName = "Mounted Radio Removable";
+			tooltip = "Determines whether radios added by default are removable.";
+			control = "CheckBox";
+			expression = "_this setVariable ['%s',_value,true];";
+			condition = "objectVehicle + (object * (1-objectBrain)) + (1 - objectControllable)";
+			defaultValue = "false";
+		};
+		class UO_FW_ACRE_WHITELISTED_POSITIONS {
+			property = "UO_FW_ACRE_WHITELISTED_POSITIONS";
+			displayName = "Whitelisted Pos.";
+			tooltip = "Postions where this rack can be accessed.";
+			control = "EditShort";
+			expression = "_this setVariable ['%s',_value,true];";
+			condition = "objectVehicle + (object * (1-objectBrain)) + (1 - objectControllable)";
+			defaultValue = "'[""driver"", [""cargo"",0]]'";
+		};
+		class UO_FW_ACRE_ADD_RACK_BLACKLISTED_POSITIONS {
+			property = "UO_FW_ACRE_BLACKLISTED_POSITIONS";
+			displayName = "Blacklisted POSITIONS";
+			tooltip = "Postions where the rack cannot be accessed.";
+			control = "EditShort";
+			expression = "_this setVariable ['%s',_value,true];";
+			condition = "objectVehicle + (object * (1-objectBrain)) + (1 - objectControllable)";
+			defaultValue = "'[""inside""]'";
+		};
+		class UO_FW_ACRE_FACTION_PRESET {
+			property = "UO_FW_ACRE_FACTION_PRESET";
+			displayName = "Faction Radio Preset";
+			tooltip = "Select the default radio presets for the object.";
+			control = "UO_FW_ACRE_Rack_FactionPreset";
+			expression = "_this setVariable ['%s',_value,true];";
+			condition = "objectVehicle + (object * (1-objectBrain)) + (1 - objectControllable)";
+			defaultValue = "west";
```

Comment:
Has this attribute value been tested? I think the attribute loading/expression might have an issue with the side vars, but that might just be the alias vars for them (`resistance` works but not `independent`), you could pass them as strings like you do in your toolbox and do a switch in your fnc

Reply:
Untested. I will be unable to do testing myself due to my schedule.